src/components/FilesRender.tsx

```
@@ -720,3 +767,170 @@ function ImagePreviewModal({
+		if (miniPlayerAudio) {
+			setMiniPlayerAudio &&
+				setMiniPlayerAudio({
+					fileData: { ...fileData, folderId: '0', date: new Date().toISOString() },
+					blobURL: miniPlayerAudio.blobURL,
+					isPlaying: false,
+					progress: miniPlayerAudio?.progress ?? 0,
+					duration: miniPlayerAudio?.duration ?? 0,
+					currentTime: miniPlayerAudio?.currentTime ?? 0,
+					isMinimized: false,
+					fileTelegramId: fileData.fileTelegramId
+				});
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)